tests/integration_tests/performance/test_network_ab.py

```diff
@@ -62,7 +62,7 @@ def test_network_latency(network_microvm, metrics):
 
     rounds = 15
     request_per_round = 30
```

Comment:
When I did my testing, I was able to put this value much higher (10k+) as the test was much faster. Maybe it's something we may want to consider

Reply:
I just cherry-picked your commit :D 

But since the test is stable enough as is, I think it's fine to leave it at 30. That's 450 data points, which is a lot for permutation testing already.

Reply:
If it's stable at 30 then sounds good!